package.json

```diff
@@ -60,7 +60,7 @@
     "url": "https://github.com/elastic/kibana.git"
   },
   "dependencies": {
-    "@elastic/eui": "3.7.0",
+    "@elastic/eui": "3.9.0",
```

Comment:
This seems to lead to some genuine test failures caused by outdated jest snapshots.

Reply:
Sorry... was an effort to try and fix EUI's typing issues.